### ISSUE TITLE
Need a way to mass-assign tags to events

### ISSUE THREAD

**Issue by GregSutcliffe** [OpenACalendar/OpenACalendar-Web-Core#602]
Related to #208 but wider in scope - being able to update tags on a range of events would be excellent.

**Comment by jarofgreen** [comment]
What do you mean by range in this case?

**Comment by GregSutcliffe** [comment]
I suppose I mean an arbitrary list of events really - which leads to the rabbit hole of being able to edit from the filter list. Perhaps I should forget it and wait for #208 :)

For reference, this came up as a result of #603 - I wanted to add a PreBook tag to a whole bunch of events.

**Comment by jarofgreen** [comment]
@GregSutcliffe For a quick start, would having a page able to edit tags on all future events in a group at once be good?

**Comment by GregSutcliffe** [comment]
Yes. But I'll add 2 caveats for interest - (1) what if the group has multiple event types? (2) why limit yourself to all future events?

**Comment by jarofgreen** [comment]
1) Yes, a fuller mass edit would certainly be good to. This is just a quick win with a simpler UI.
2) I was just saying future as distinct from past events, don't see much point editing tags on past events.

**Comment by GregSutcliffe** [comment]
1) That's fair, sure
2) I guess the future-only limitation implied some form of filtering, which leads to the question of "well why not go the whole hog then?". But I agree, there's not much point altering past events.

**Comment by jarofgreen** [comment]
2) Because I can hard code that filter as a "common sense" one, but anything else would involve working out some form of UI/UX. We'll tackle that later.
